app/mover.js

```diff
@@ -165,6 +166,11 @@ class Mover {
 
       // If its dropped, then bail
       if (currentReturn < this.config.sell_at_loss_percent) {
+        sell_at_loss_count++
+        if (sell_at_loss_count < this.config.sell_at_loss_count) {
```

Comment:
@ChrisCPoirier Should this be `<=` ?

Reply:
@thisiskj I do not believe so, The intent is that if the sell_at_loss_count is below the config.sell_at_loss_count we `continue` the main loop and recheck prices, else we have matched or exceed our threshold and want to move forward with the sell